### PR TITLE
fixes #7062: facts module uses correct path for 'requires'

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: modules/facts/facts.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 include ::Proxy::Log
 begin
   require "facter"
-  require 'facts_plugin'
+  require 'facts/facts_plugin'
```
Comment (domcleal) [inline modules/facts/facts.rb]: Maybe it's better to record if loading 'facter' was successful, and only require facts_plugin if it was known to be OK, so we don't mask errors in the loading of facts_plugin.

Comment (dmitri-d) [inline modules/facts/facts.rb]: yeah, good point.

**Comment by dmitri-d** [comment]
@domcleal: ping?

**Comment by lzap** [comment]
I am still testing this.

**Comment by lzap** [comment]
And it works in discovery image. :+1: Thanks for prompt fix @witlessbird !

**Comment by lzap** [comment]
Any chance getting this merged so I have it in tomorrows nightly?

**Comment by dmitri-d** [comment]
merged in at 4bfe725

**Pull request closed without merging**